help/license.md

```diff
@@ -1,103 +1,85 @@
-arXiv License Information
-=========================
+# arXiv License Information
+
+<style>
+.mkd-img-left {
+  float:left;
+  width:10%;
+  margin-top:0;
+  padding-right:5px;
+}
+blockquote {
+  border-left: 0;
+  -webkit-box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  -moz-box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  box-shadow: 0px 3px 8px 0px rgba(0,0,0,0.12);
+  padding:1em;
+  margin-bottom:1.5em;
+}
+@media (min-width: 576px) {
+  blockquote {
+    padding: 2em;
+  }
+}
+</style>
+
+As a repository for scholarly material, arXiv keeps a permanent record of every article and version posted. All articles on arXiv.org can be viewed and downloaded freely by anyone.
+
+To maintain this scholarly record, arXiv needs permission from submitters to post and distribute their work. Permission is granted through a license, chosen by the submitter, and the submitter must certify that they have the right to grant that license.
+
+Several types of licenses are available, listed below. For all licenses here, with the exception of CC Zero, the original copyright holder retains ownership after posting on arXiv.
+
+How to choose a license
+-----------------------
+
+arXiv values open access and encourages choosing a liberal license for re-use of the paper. However, each situation is unique. Funders may require specific licenses. Journals have different policies regarding preprints, embargo periods, and post-publication distribution. Please note that different versions of the work can have different licenses.
+
+Ultimately, it is the submitter’s responsibility to choose the appropriate license for their content. The license chosen is irrevocable and cannot be changed.
+
+To make a license decision, arXiv recommends the following:
+1. Read and understand the [submission terms and agreement](policies/submission_agreement) and [instructions for submission](policies/instructions_for_submission)
+2. Understand the meanings and implications of each license below
+3. Check the policies of funders and journals (contact journals directly or using a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html))

```

Comment:
Check the policies of funders and journals (contact journals directly or use a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html))